src/Microsoft.FeatureManagement.AspNetCore/TagHelpers/FeatureTagHelper.cs

```diff
@@ -52,11 +59,35 @@ public override async Task ProcessAsync(TagHelperContext context, TagHelperOutpu
 
             if (!string.IsNullOrEmpty(Name))
             {
-                IEnumerable<string> names = Name.Split(',').Select(n => n.Trim());
+                IEnumerable<string> features = Name.Split(',').Select(n => n.Trim());
+
+                if (string.IsNullOrEmpty(Variant))
+                {
+                    enabled = Requirement == RequirementType.All ?
+                        await features.All(async feature => await _featureManager.IsEnabledAsync(feature).ConfigureAwait(false)) :
+                        await features.Any(async feature => await _featureManager.IsEnabledAsync(feature).ConfigureAwait(false));
+                }
+                else
+                {
+                    if (features.Count() != 1)
+                    {
+                        throw new ArgumentException("Variant cannot be associated with multiple feature flags.", nameof(Name));
+                    }
+
+                    IEnumerable<string> variants = Variant.Split(',').Select(n => n.Trim());
+
+                    if (variants.Count() != 1 && Requirement == RequirementType.All)
```

Comment:
This check goes against the "Have no effect" we said in the readme. We should either always assume RequirementType.Any or we should throw. I believe we aligned on throwing.

Reply:
I remember I have removed "Have no effect" in readme.

Reply:
Updated.